
########################################  Running CUTe test(s)  #########################################

expect_uint_suite::expect_uint_caller_fail_test .................................................... fail
issue:  exec failed
reason: mock caller function mismatch
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:26
    caller: nonexistent_callee()
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()

expect_uint_suite::expect_uint_parm_fail_test ...................................................... fail
issue:  exec failed
reason: unsigned integer mock parameter name mismatch
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:37
    caller: expect_uint_value_callee()
    expect: nonexistent_parm == 2
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:11
    caller: expect_uint_value_callee()
    expect: word == 2

expect_uint_suite::expect_uint_literal_equal_fail_test ............................................. fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:69
    caller: expect_uint_value_callee()
    expect: byte == 1
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] != [1]

expect_uint_suite::expect_uint_var_equal_fail_test ................................................. fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:77
    caller: expect_uint_value_callee()
    expect: byte == bref
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] != [1]

expect_uint_suite::expect_uint_literal_unequal_fail_test ........................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:111
    caller: expect_uint_value_callee()
    expect: byte != 0
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] == [0]

expect_uint_suite::expect_uint_var_unequal_fail_test ............................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:119
    caller: expect_uint_value_callee()
    expect: byte != bref
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] == [0]

expect_uint_suite::expect_uint_literal_greater_fail_test ........................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:159
    caller: expect_uint_value_callee()
    expect: byte > 1
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] <= [1]

expect_uint_suite::expect_uint_var_greater_fail_test ............................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:167
    caller: expect_uint_value_callee()
    expect: byte > bref
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] <= [1]

expect_uint_suite::expect_uint_literal_greater_equal_fail_test ..................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:219
    caller: expect_uint_value_callee()
    expect: byte >= 1
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] < [1]

expect_uint_suite::expect_uint_var_greater_equal_fail_test ......................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:227
    caller: expect_uint_value_callee()
    expect: byte > bref
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] <= [1]

expect_uint_suite::expect_uint_literal_lower_fail_test ............................................. fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:261
    caller: expect_uint_value_callee()
    expect: byte < 0
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] >= [0]

expect_uint_suite::expect_uint_var_lower_fail_test ................................................. fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:269
    caller: expect_uint_value_callee()
    expect: byte < bref
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [1] >= [0]

expect_uint_suite::expect_uint_literal_lower_equal_fail_test ....................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:309
    caller: expect_uint_value_callee()
    expect: byte <= 0
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [1] > [0]

expect_uint_suite::expect_uint_var_lower_equal_fail_test ........................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:317
    caller: expect_uint_value_callee()
    expect: byte <= bref
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [1] > [0]

expect_uint_suite::expect_uint_literal_in_range_fail_test .......................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:353
    caller: expect_uint_value_callee()
    expect: byte in range {1 ... 10}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] not in range {1 ... 10}

expect_uint_suite::expect_uint_var_in_range_fail_test .............................................. fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:365
    caller: expect_uint_value_callee()
    expect: byte in range {rmin ... rmax}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] not in range {1 ... 10}

expect_uint_suite::expect_uint_literal_not_in_range_fail_test ...................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:399
    caller: expect_uint_value_callee()
    expect: byte not in range {5 ... 9}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [5] in range {5 ... 9}

expect_uint_suite::expect_uint_var_not_in_range_fail_test .......................................... fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:411
    caller: expect_uint_value_callee()
    expect: byte not in range {rmin ... rmax}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [5] in range {5 ... 9}

expect_uint_suite::expect_uint_literal_in_set_fail_test ............................................ fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:444
    caller: expect_uint_value_callee()
    expect: byte in set {1, 5, 10, 15}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] not in set {1, 5, 10, 15}

expect_uint_suite::expect_uint_var_in_set_fail_test ................................................ fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:457
    caller: expect_uint_value_callee()
    expect: byte in set {ref0, 5, ref2, 15}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [0] not in set {1, 5, 10, 15}

expect_uint_suite::expect_uint_literal_not_in_set_fail_test ........................................ fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:491
    caller: expect_uint_value_callee()
    expect: byte not in set {1, 5, 10, 15}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [1] in set {1, 5, 10, 15}

expect_uint_suite::expect_uint_var_not_in_set_fail_test ............................................ fail
issue:  exec failed
reason: unsigned integer mock parameter check failed
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:504
    caller: expect_uint_value_callee()
    expect: byte not in set {ref0, 5, ref2, 15}
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:9
    caller: expect_uint_value_callee()
    actual: [1] in set {1, 5, 10, 15}

expect_uint_suite::expect_uint_retval_caller_fail_test ............................................. fail
issue:  exec failed
reason: mock caller function mismatch
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:540
    caller: nonexistent_callee()
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:512
    caller: expect_uint_retval_callee()

expect_uint_suite::expect_xcess_uint_retval_fail_test .............................................. fail
issue:  teardown failed
reason: extra mock expectation left
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:553
    caller: expect_uint_retval_callee()
    expect: mock function return value
found:  none

expect_uint_suite::expect_uint_inval_retval_fail_test .............................................. fail
issue:  exec failed
reason: mock expectation type mismatch
wanted:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:563
    caller: expect_uint_retval_callee()
    expect: mock function parameter
found:
    source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:512
    caller: expect_uint_retval_callee()
    found:  mock function return value

expect_uint_suite .................................................................................. fail
issue:  exec failed
reason: descendants failed
source: /mnt/devel/greg/tidor/cute/test/expect_uint.c:620

NAME                                           STAT  TIME(s.us)  #PASS  #SKIP  #FAIL  #EXCP  #EXEC #TOTAL
expect_uint_suite                              fail    0.852029     11      0     25      0     36     36
                                                                   30%     0%    69%     0%   100%
---------------------------------------------------------------------------------------------------------
Total                                          fail    0.852029     11      0     25      0     36     36
                                                                   30%     0%    69%     0%   100%

                                  ###           SUMMARY            ###
                                  ###  25/36 (69%) test(s) FAILED  ###

